### ISSUE TITLE
Timestamp' object has no attribute 'split' problem

### ISSUE THREAD

**Issue by DanananBananan** [pandas-dev/pandas#33181]
import pandas as pd
import numpy as np
from sklearn.svm import SVR
from sklearn.linear_model import LinearRegression
import matplotlib.pyplot as plt

df1 = pd.read_excel("npl90.xlsx",parse_dates=True)
dates = []
npl = []
df1.shape
df=df1.iloc[::-1]

df.tail(1)
df=df.head(len(df)-1)
df
df_dates=df.loc[:, "Date"]
df_npl=df.loc[:, 'NPL90+']

#Create independent data set x
for date in df_dates:
    dates.append([int(date.split('-')[1])])

def predict_npl(dates, npl, x):
    svr_lin= SVR(kernel="linear", C=1e3)
    svr_poly= SVR(kernel="poly", C=1e3, degree=2)
    svr_rbf=SVR(kernel="rbf", C=1e3, gamma=0.1)
    #Training
    svr_poly.fit(df_dates, df_npl)
    svr_lin.fit(df_dates, df_npl)
    svr_rbf.fit(df_dates, df_npl)
    
    #Create the linear regression
    lin_reg=LinearRegression()
    #Train the Linear regression model
    lin_reg.fit(df_dates, df_npl)
    
    
    #Plot models
    plt.scatter(df_dates, df_npl, color="black", label="original data")
    plt.plot(df_dates, svr_rbf.predict(df_dates), color='red', label= 'SVR RBF')
    plt.plot(df_dates, svr_poly.predict(df_dates), color='blue', label= 'SVR POLY')
    plt.plot(df_dates, svr_lin.predict(df_dates), color='green', label= 'SVR LIN')
    plt.plot(df_dates, lin_reg.predict(df_dates), color='grey', label= 'LIN Reg')
    plt.xlabel('Date')
    plt.ylabel('NPL')
    plt.title('Regression')
    plt.legend()
    plt.show()
    
    return svr_rbf.predict(x)[0], svr_lin.predict(x)[0], svr_poly.predict(x)[0], lin_reg.predict(x)[0]
    
#Predict price on 
predicted_npl=predict_npl(dates, npl,"2020-03-01")
print(predicted_npl)

[npl90.xlsx](https://github.com/pandas-dev/pandas/files/4409483/npl90.xlsx)

**Comment by mroeschke** [comment]
I Imagine `df_dates=df.loc[:, "Date"]` is not `str` type but `datetime64[ns]` type. If you want to format the dates you can use `dt.strftime` on that column.

For usage questions like these, we recommend using StackOverflow
